frontend/src/scenes/error-tracking/errorTrackingIssueSceneLogic.ts

```diff
@@ -132,46 +108,101 @@ export const errorTrackingIssueSceneLogic = kea<errorTrackingIssueSceneLogicType
                 errorTrackingIssueEventsQuery({
-                    issue,
+                    issueId,
                     filterTestAccounts: filterTestAccounts,
                     filterGroup: filterGroup,
                     dateRange,
                 }),
         ],
 
-        issueProperties: [
+        issueDateRange: [(s) => [s.issue], (issue) => (issue?.first_seen ? getIssueDateRange(issue) : {})],
+
+        firstSeen: [
             (s) => [s.issue],
-            (issue): Record<string, any> => (issue && issue.earliest ? JSON.parse(issue.earliest) : {}),
+            (issue: ErrorTrackingRelationalIssue | null) => (issue?.first_seen ? dayjs(issue.first_seen) : null),
         ],
     }),
 
-    listeners(({ values, actions }) => {
-        const loadIssue = (): void => {
-            if (!values.issueLoading) {
-                const issue = values.issue
-                if (!issue) {
-                    actions.loadRelationalIssue()
-                } else {
-                    actions.loadClickHouseIssue(dayjs(issue.first_seen))
-                }
-            }
-        }
+    loaders(({ actions, props }) => ({
+        issue: {
+            loadIssue: async () => {
+                const issue = await api.errorTracking.getIssue(props.id, props.fingerprint)
+                return issue
+            },
+        },
+        properties: [
+            {} as Record<string, string>,
+            {
+                loadProperties: async ({ issue }) => {
+                    const firstSeen = dayjs(issue.first_seen)
+                    const response = await api.query(
+                        errorTrackingIssueQuery({
+                            issueId: props.id,
+                            dateRange: {
+                                date_from: firstSeen.startOf('hour').toISOString(),
+                                date_to: firstSeen.endOf('hour').toISOString(),
+                            },
+                            volumeResolution: 0,
+                        }),
+                        {},
+                        undefined,
+                        'blocking'
+                    )
+                    const issueAgg = response.results[0]
+                    return JSON.parse(issueAgg.earliest!)
+                },
+            },
+        ],
+        summary: {
+            loadSummary: async ({ dateRange }) => {
+                const response = await api.query(
+                    errorTrackingIssueQuery({
+                        issueId: props.id,
+                        dateRange,
+                        volumeResolution: 40,
+                    }),
+                    {},
+                    undefined,
+                    'blocking'
+                )
+                const issue = response.results[0]
+                actions.setSummary(issue.last_seen!, JSON.parse(issue.earliest!), issue.aggregations!)
+                return null
```

Comment:
We should not be calling `setSummary` and returning null here... that is what the loader is for 
```suggestion
                return { lastSeen: issue.last_seen, aggregations: issue.aggregations }
```